### PR TITLE
Issue 1213

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
for: https://github.com/openstax/unified/issues/1213

### CODE REVIEW

File: src/app/content/highlights/hooks/loadMore.spec.ts [hunk 1]
```diff
@@ -130,6 +132,7 @@ describe('filtersChange', () => {
       }))
     ;
 
+    filters = store.getState().content.highlights.summary.filters;
```
Comment (janKozie1) [inline src/app/content/highlights/hooks/loadMore.spec.ts:135]: Do the filters get changed somewhere?

File: src/app/content/highlights/reducer.spec.ts [hunk 2]
```diff
@@ -406,6 +407,36 @@ describe('highlight reducer', () => {
...
+          ...initialState.summary,
+          filters: {
+            colors: [HighlightColorEnum.Green],
+            locationIds: ['id'],
+          },
+          loading: true,
+        },
+      };
+
+      const staleFilters = {} as any as SummaryFilters;
```
Comment (janKozie1) [inline src/app/content/highlights/reducer.spec.ts:432]: You could add `locationIds: [], colors: []` into the object and it wouldn't need to be casted.

**Review by janKozie1 — COMMENTED** [review]
You could write a test with timeouts if you wanted to replicate the problem. 
Left some minor comments too.

**Review by janKozie1 — APPROVED** [review]

File: src/app/content/highlights/reducer.ts [hunk 3]
```diff
@@ -237,6 +238,10 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
       };
     }
     case getType(actions.receiveSummaryHighlights): {
+      // Check if filters wasn't updated while we were loading response.
+      // It may happen if user with slow network connection change filters very fast.
+      if (action.meta.filters && !equals(state.summary.filters, action.meta.filters)) { return state; }
```
Comment (TomWoodward) [inline src/app/content/highlights/reducer.ts:243]: you can use strict equality here, i don't think lodash equals is necessary

**Review by TomWoodward — CHANGES_REQUESTED** [review]

**Review by TomWoodward — APPROVED** [review]

**Pull request merged**